app/components/channel_drawer/channels_list/list/list.js

```diff
@@ -218,6 +227,25 @@ class List extends PureComponent {
         this.setState({width: width - 40});
     };
 
+    onPreview = (item) => {
+        if (Platform.OS === 'ios') {
+            const {intl} = this.context;
+
+            this.props.navigator.push({
+                screen: 'ChannelPeek',
+                previewCommit: false,
```

Comment:
What is this prop used for?

Reply:
To tell react native navigation that we do not want the preview component to be added to the navigation stack